instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java

```diff
@@ -72,6 +84,19 @@ public Builder remoteServiceName(String remoteServiceName) {
       return this;
     }
 
+    /**
+     * Controls the sharing of a poll span for incoming spans with no trace context.
+     *
+     * <b/>If true, all the spans received in a poll batch that do not have trace-context will be added
```

Comment:
Might be good to use `{@code poll}` to highlight it as a keyword, not an English word.